test/addons-napi/test_reference/test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+// Flags: --expose-gc
+
+const common = require('../../common');
+const assert = require('assert');
+
+const test_reference = require(`./build/${common.buildType}/test_reference`);
```

Comment:
The underscore matches the name of the module. I was trying to follow precedent in node.js JavaScript code, often with the `child_process` module, for example at https://github.com/nodejs/node/blob/master/test/common.js#L28